crates/server/src/store/recursor.rs

```diff
@@ -234,6 +242,14 @@ pub struct RecursiveConfig {
     #[serde(default)]
     pub dnssec_policy: DnssecPolicyConfig,
 
+    /// Networks that will be queried during resolution
```

Comment:
is this rustdoc comment (and the one on L249) correct? I thought these were being used for answer filtering, not query filtering.